### DISCUSSION TITLE
Hi! It it possible to hash css classes? specially for building the web app?

### DISCUSSION THREAD

**Discussion by JenuelDev** [unocss/unocss#2996] [Q&A] [answered]
So, currently, there are a lot of things happening right now in the world of css, but It seems that a lot of websites are now hashing their css classes. Every time I open dev tools and check elements, they have this obscure or hashed css classes.

This is useful to avoid people manipulating the website using an chrome extensions or any css overidings.

![image](https://github.com/unocss/unocss/assets/31676163/819b9b95-0e54-4132-ab8d-d2607ec1e87d)

**Comment by Shyam-Chen** [comment, marked as answer]
https://github.com/unocss/unocss/tree/main/packages/transformer-compile-class

**Reply by JenuelDev** [reply]
ok, I see that it has this options.. What if the same classes used elswhere?

**Reply by zyyv** [reply]
FYI, https://github.com/unpreset/unocss-transformer-hash  but I'm not sure how well it works, you can try.
